docs/schedule/examples/flex.md

```diff
@@ -0,0 +1,326 @@
+<a class="pencil-link" href="https://github.com/MobilityData/gtfs.org/edit/main/docs/schedule/examples/flex.md" title="Edit this page" target="_blank">
+    <svg class="pencil" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M10 20H6V4h7v5h5v3.1l2-2V8l-6-6H6c-1.1 0-2 .9-2 2v16c0 1.1.9 2 2 2h4v-2m10.2-7c.1 0 .3.1.4.2l1.3 1.3c.2.2.2.6 0 .8l-1 1-2.1-2.1 1-1c.1-.1.2-.2.4-.2m0 3.9L14.1 23H12v-2.1l6.1-6.1 2.1 2.1Z"></path></svg>
+  </a>
+
+# Flex
+
+<hr>
+
+Flex is a GTFS extension project that aims to facilitate discoverability of Demand Responsive Transportation Services.
+
+The following example demonstrates how to model different demand responsive service use cases using Flex. **Please note that the following examples are not necessarily an accurate or complete representation of the agencies’ services.**
+
+## On-demand services within a single zone
+
+Demand-repsonsive services can operate within a specific zone, allowing riders to book pickups at any point A within the zone and drop-offs at any point B within the same zone. An example of this is the [Heartland Express Transit](https://www.co.brown.mn.us/heartland-express-transit?view=category&id=56) service in Minnesota, USA.
+
+<sup>[Download Heartland Express example dataset](../../../assets/on-demand_services_within_a_single_zone.zip)</sup>
+
+### Define trips
+
+Heartland Express service hours are as follow:
+
+- Weekdays:
+  - 8:00 AM - 5:00 PM
+  - 6:15 AM – 5:45 PM (New Ulm Only)
+- Sunday: 8:00 AM - Noon (New Ulm Only)
+
+The New Ulm city zone is included within the Brown County zone. To avoid the "zone overlap constraint" issue, Heartland Express can be defined with four trips:
+
+- Service in New Ulm from 6:15 AM to 8:00 AM on weekdays.
+- County-wide service from 8:00 AM to 5:00 PM on weekdays.
+- Service in New Ulm from 5:00 PM to 5:45 PM on weekdays.
+- Service in New Ulm from 8:00 AM to 12:00 PM on Sundays.
+  
+[**trips.txt**](../../reference/#tripstxt)
+
+route_id | service_id | trip_id
+-- | -- | -- 
+74362 | c_67295_b_77497_d_31 | t_5374945_b_77497_tn_0
+74362 | c_67295_b_77497_d_31 | t_5374946_b_77497_tn_0
+74362 | c_67295_b_77497_d_31 | t_5374944_b_77497_tn_0
+74362 | c_67295_b_77497_d_64 | t_5374947_b_77497_tn_0
+
+`service_id = c_67295_b_77497_d_31` refers to weekdays, `service_id = c_67295_b_77497_d_64` refers to Sunday.
+
+### Define zones (GeoJSON locations)
```

Comment:
We're in the "On-demand services within a single zone" section, maybe this should be "Define zone" instead? (in the singular form)

Reply:
Updated